docker/e2e/cypress/integration/migrating_users.spec.js

```diff
@@ -90,10 +90,32 @@ describe('user migration plugin', () => {
     }
 
     function configureEmails() {
+        writeAdminPersonalInfo();
```

Comment:
Are you sure this is needed for the tests? I just ran them without invoking this method and they all passed.